react-components/toolbar-filter/step2.md

```diff
@@ -1,61 +1,141 @@
-At its most basic, Toolbars consist of content rows of components. Most toolbars will only be a single row, but each row needs to be housed in a `DataToolbarContent` component.
-
-## Task
-1) Navigate to the `src` folder and open `src/App.js`{{open}}
-
-Next, we'll set up the general structure of an empty toolbar and add the necessary imports.
-
-2) Copy the following code into the App.js file, replacing all of the content there:
-
-<pre class="file" data-filename="App.js" data-target="replace">
-import React from &quot;react&quot;;
-import &quot;@patternfly/react-core/dist/styles/base.css&quot;;
-import { Button } from &quot;@patternfly/react-core&quot;;
-import {
-  EditIcon,
-  CloneIcon,
-  SyncIcon,
-  FilterIcon
-} from &quot;@patternfly/react-icons&quot;;
-import {
-  DataToolbar,
-  DataToolbarContent,
-  DataToolbarItem,
-  DataToolbarGroup,
-  DataToolbarToggleGroup,
-  DataToolbarFilter
-} from &quot;@patternfly/react-core/dist/esm/experimental&quot;;
-import Input from &quot;./components/input&quot;;
-import CheckboxSelect from &quot;./components/checkboxSelect&quot;;
-import Kebab from &quot;./components/kebab&quot;;
-import AppTable from &quot;./components/table&quot;;
-
-class ToolbarDemoApp extends React.Component {
-  constructor(props) {
-    super(props);
-  }
-
-  render() {
-    return (
-      &lt;React.Fragment&gt;
-        Toolbar Demo
-      &lt;/React.Fragment&gt;
-    );
-  }
+The individual components placed inside of a toolbar are each wrapped in a `DataToolbarItem` component. `DataToolbarItems` can be passed an optional `variant` prop to format a specific type of component it wraps such as a search filter, a pagination component, a bulk select, or an overflow menu. 
```

Comment:
```suggestion
The individual components placed inside of a toolbar are each wrapped in a `DataToolbarItem` component. `DataToolbarItems` can be passed an optional `variant` prop to format a specific type of component that it. `DataToolbarItems` wrap components like the search filter, pagination component, bulk select, or overflow menu. 
```

Reply:
The grammar in this suggestion seems off.
I'll change it to: The individual components placed inside of a toolbar are each wrapped in a `DataToolbarItem` component. `DataToolbarItems` can be passed an optional `variant` prop to format a specific type of component it wraps. These specific components could be search filter, a pagination component, a bulk select, or an overflow menu.